packages/create-cloudflare/e2e-tests/frameworks.test.ts

```diff
@@ -74,6 +189,80 @@ function getFrameworkTests(opts: {
 				},
 				unsupportedOSs: ["win32"],
 			},
+			nuxt: {
+				quarantine: true,
+				testCommitMessage: true,
+				timeout: LONG_TIMEOUT,
+				unsupportedOSs: ["win32"],
+				verifyDeploy: {
+					route: "/",
+					expectedText: "Welcome to Nuxt!",
+				},
+				verifyPreview: {
+					route: "/test",
+					expectedText: "C3_TEST",
+				},
+				verifyBuildCfTypes: {
+					outputFile: "worker-configuration.d.ts",
+					envInterfaceName: "Env",
+				},
+			},
+			solid: {
+				promptHandlers: [
+					{
+						matcher: /Which template would you like to use/,
+						input: [keys.enter],
+					},
+					{
+						matcher: /Use Typescript/,
+						input: [keys.enter],
+					},
+				],
+				testCommitMessage: true,
+				timeout: LONG_TIMEOUT,
+				unsupportedPms: ["npm", "yarn"],
+				unsupportedOSs: ["win32"],
+				verifyDeploy: {
+					route: "/",
+					expectedText: "Hello world",
+				},
+				verifyPreview: {
+					route: "/",
+					expectedText: "Hello world",
+				},
+			},
+			svelte: {
+				promptHandlers: [
+					{
+						matcher: /Which template would you like/,
+						input: [keys.enter],
+					},
+					{
+						matcher: /Add type checking with Typescript/,
+						input: [keys.down, keys.enter],
+					},
+					{
+						matcher: /What would you like to add to your project/,
+						input: [keys.enter],
+					},
+					{
+						matcher:
+							/Which package manager do you want to install dependencies with/,
+						input: [keys.enter],
+					},
+				],
+				testCommitMessage: true,
+				unsupportedOSs: ["win32"],
```

Comment:
just noticed no windows tests actually end up running 🤣 only gatsby supports windows, but we run pnpm for the windows test and gatsby doesn't support that. not sure if worth changing that the windows PM to npm or something.

Reply:
We should tidy that up as a follow up

Reply:
https://github.com/cloudflare/workers-sdk/issues/7412